articles/managed-operations/man-how-configure-vpn-for-openstack.md

```diff
@@ -0,0 +1,298 @@
+
+   ![Security group rules for SSH & Web Access security group](images/man-mmaas-openstack-vpn-security-group2.png)
+
+6. Create a third security group with the name `Managed Monitoring Ports`.
+
+   - The rules for this security group allow the UKCloud IT Operations subnet to communicate with the ports that are required to be monitored as specified in the MITO Data Capture Form. You must also allow ICMP to and from the local and peer subnets.
+
+   - The Ingress entry IP address must be the UKCloud IT Operations subnet, which will be provided by UKCloud.
+
+   - The Egress entry IP address must be the required monitored subnet(s), which are specified in the MITO Data Capture form.
+
+   ![Security group rules for Managed Monitoring Ports security group](images/man-mmaas-openstack-vpn-security-group3.png)
+
+7. If you have opted for UKCloud Patching as a Service you must also create an additional security group with the name `Managed Patching Ports`.
+
+   The rules for this security group allow the UKCloud IT Operations subnet to communicate through the VPN to patch the machines on the patching ports. Raise a service request to obtain the ports that require opening.
+
+### Deploying the VPN instance
+
+You're now ready to deploy the instance.
+
+1. Expand **Compute** then select **Instances**.
+
+2. Click the **Launch Instances** button.
+
+3. In the *Launch Instance* dialog box, provide the following information:
+
+   - **Details** tab:
+
+     - **Instance Name:** Enter `MMaaS-VPN-Endpoint` or similar.
+
+     - **Description:** Leave blank.
+
+     - **Availability Zone:** Select **Any Availability Zone**.
+
+     - **Count:** Enter `1`.
+
+   - **Source** tab:
+
+     - **Select Boot Source:** Select **Image**.
+
+     - **Create New Volume:** Select **No**.
+
+     - Select your pfSense image from the list of images.
+
+   - **Flavor** tab:
+
+     - Select **t1.large**.
+
+   - **Networks** tab:
+
+     - Select the network that you created earlier.
+
+   - **Security Groups** tab:
+
+     - Select the security groups that you created earlier: UKCloud IT Operations VPN, SSH & Web Access, Managed Monitoring Ports, Managed Patching Ports (if applicable).
+
+4. Click **Launch Instance**.
+
+5. On the *Instances* page, in the dropdown for your new instance, select **Associate Floating IP**.
+
+6. You'll need to add two **Allowed Address Pairs** entries for the instance's port.
+
+   - In the Horizon dashboard, expand **Networks** then select **Network**.
+
+   - Click your VPN network's name.
+
+   - Select the **Ports** tab.
+
+   - Click the port that corresponds to the instance's internal IP address.
+
+   - Click **Allowed Address Pairs** then **+**.
+
+   - Create an address pair for the local subnet and click **Submit**.
+
+   - Create a second address pair for the peer subnet and click **Submit**.
+
+7. You need to add a static route to the OpenStack router to ensure that traffic for the Opsview collectors are routed via the VPN.
```

Comment:
Dont mention Opsview collectors - please change this to "UKCloud Monitoring infrastructure is routed via the VPN"

Reply:
Fixed